test/cmd.js

```diff
@@ -895,6 +895,60 @@ describe('express(1)', function () {
         .expect(404, /<h1>Not Found<\/h1>/, done)
       })
     })
+
+    describe('--view=none', function () {
```

Comment:
Oh, oops, the build output was weird. Yea, it should be in alpha order like the test of the tests in the section and just named `none`, i.e. the section is the name of what is filled into `<engine>` placeholder.